examples/remix-with-typescript/app/src/theme.ts

```diff
@@ -14,6 +14,13 @@ const theme = createTheme({
       main: red.A400,
     },
   },
+  components: {
+    MuiButton: {
+      defaultProps: {
+        color: "secondary"
+      }
+    }
+  }
```

Comment:
```suggestion
```

I guess this was added for testing purposes. My recommendation is to remove it, as for someone cloning the example as a starter project, this is not something they would expect to come along the way :)